SEP-0006.md

```diff
@@ -39,14 +39,14 @@ table lists the start and end of individual appointments.
 
 # Board Roles
 Board roles are 1-year long appointments. The following information shows
-the start and end of individual appointment.
+the start and end of individual appointments.
 
 | Role                    | Name                        | Start       | Expiration  |
 |-------------------------|-----------------------------|-------------|-------------|
-| Chair                   | Steven Christe              | 31 Dec 2017 | 31 Dec 2018 |
-| Vice-chair              | David Perez-Suarez          | 31 Dec 2017 | 31 Dec 2018 |
-| Chief Financial Officer | David Perez-Suarez (acting) | 31 Dec 2017 | 31 Dec 2018 |
-| Secretary               | Sabrina Savage              | 31 Dec 2017 | 31 Dec 2018 |
+| Chair                   | Steven Christe              | 30 Apr 2019 | 30 Apr 2020 |
+| Vice-chair              | Monica Bobra                | 30 Apr 2019 | 30 Apr 2020 |
+| Chief Financial Officer | David Perez-Suarez          | 30 Apr 2019 | 30 Apr 2020 |
```

Comment:
Just Financial Officer? Given there is only one?

Reply:
Why don't you want me to be a chief? :disappointed: